WorkloadManagementSystem/DB/JobElasticDB.py

```diff
@@ -0,0 +1,128 @@
+
+    self.siteClient = SiteStatus()
+
+    self.log.info("==================================================")
+
+#############################################################################
+  def getJobParameters(self, jobID, paramList=None):
+    """ Get Job Parameters defined for jobID.
+      Returns a dictionary with the Job Parameters.
+      If parameterList is empty - all the parameters are returned.
+    """
+
+    self.log.debug('JobDB.getParameters: Getting Parameters for job %s' % jobID)
+
+    resultDict = {}
+
+    if paramList:
+      paramNameList = []
+
+      for x in paramList:
+        paramNameList.append(x)
+
+      query = {
+          "query": {
+              "bool": {
+                  "must": [
+                      {"match": {"JobID": jobID}}, {"match": {"Name": ','.join(paramNameList)}}]}},
+          "_source": ["Name", "Value"]}
+
+    else:
+      query = {"query": {"match": {"JobID": jobID}}, "_source": ["Name", "Value"]}
+
+    result = self.query('jobelasticdb*', query)
+
+    if result['OK']:
+      if result['Value']:
+
+        sources = result['Value']['hits']['hits']
+
+        for i in xrange(0, len(sources)):
+
+          name = sources[i]['_source']['Name']
+          value = sources[i]['_source']['Value']
+
+          try:
+            resultDict[name] = value.tostring()
+          except BaseException:
+            resultDict[name] = value
+
+        return S_OK(resultDict)
+
+    else:
+      return S_ERROR('JobDB.getJobParameters: failed to retrieve parameters')
+
+#############################################################################
+  def setJobParameter(self, jobID, key, value):
+    """ Set a parameter specified by name,value pair for the job JobID
+    """
+
+    query = {
+        "query": {
+            "term": {
+                "JobID": jobID}}, "script": {
+            "inline": "ctx._source.Value = params.value; ctx._source.Name = params.name", "params": {
+                "value": value, "name": key}}}
+    indexName = self.generateFullIndexName('jobelasticdb')
+    result = self.exists(indexName)
+
+    if not result:
+      mapping = {
+          "JobParameters": {
+              "properties": {
+                  "JobID": {
+                      "type": "long"}, "Name": {
+                      "type": "text"}, "Value": {
+                      "type": "binary"}}}}
+      result = self.createIndex('jobelasticdb', mapping)
+
+      if not result['OK']:
+        return result
+
+    result = self.update('jobelasticdb*', 'JobParameters', query)
+
+    if not result['OK']:
+      return S_ERROR('JobDB.setJobParameter: operation failed.')
+
+    if result['Value']['updated'] == 0:
+
+      query = {"JobID": jobID, "Name": key, "Value": value}
+      result = self.update(indexName, 'JobParameters', query, update_by_query=False, id=jobID)
+
+    if not result['OK']:
+      result = S_ERROR('JobDB.setJobParameter: operation failed.')
+
+    return result
```

Comment:
In both the methods above (and in this module in general) there's no use of `gLogger`. It would be useful to have at least some `gLogger.debug` messages.